docs/ssh_over_local_net.rst

```diff
@@ -0,0 +1,78 @@
+SSH Over Local Network
+======================
+
+Under a production installation post-install, the default way to gain ssh
+administrative access is over the tor network. This provides a number of benefits:
+
+* Allows remote administration outside of the local network
+* Provides anonymity to an administrator while logging into the SecureDrop
+  back-end.
+* Can mitigate against an attacker on your local network attempting to exploit
+  vulnerabilities against the ssh daemon.
+
+Most administrators will need ssh access during the course of running a
+SecureDrop instance and a few times a year for maintanence. So the
+potential short-falls of having ssh over tor aren't usually a big deal.
```

Comment:
Docs look great :) One addition: it looks like the comments [here](https://github.com/freedomofpress/securedrop/pull/2592#discussion_r153304245) need addressing